src/tool_getparam.c

```diff
@@ -565,7 +565,7 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
                             struct OperationConfig *config)
 {
   char letter;
-  char subletter = '\0'; /* subletters can only occur on long options */
+  char subletter; /* subletters can only occur on long options */
```

Comment:
I don't think we should remove "safety initializations" like this.

Reply:
ACK